learn/hoon/hoon-tutorial/recursion.udon

```diff
@@ -0,0 +1,200 @@
+:-  :~  navhome/'/docs/'
```

Comment:
@baudtack this file needs TOML metadata – we don't use sail anymore. See other documents in the repo for a template, and note the "weight" key, which specifies the sort order in the sidebar